### DISCUSSION TITLE
Codespaces terminal is ugly :(

### DISCUSSION THREAD

**Discussion by EduApps-CDG** [community/community#20978] [Codespaces] [answered]
Take a look:
![image](https://user-images.githubusercontent.com/32627482/180105204-9db81322-5b9c-4597-9248-bd2911acbd33.png)
I can't even read!

**Comment by thaisonzdx** [comment]
You can change terminal font in Settings at`Terminal > Intergrated > Font family` section. Using search bar to find it quickly.

**Reply by EduApps-CDG** [reply]
Works with `Sans serif` but with `monospace` or `Ubuntu Mono` it gets ugly.

**Comment by DenverCoder1** [comment, marked as answer]
If you are using Brave or a similar privacy browser, it is likely because of the fingerprint blocking features that prevent the browser from checking for available fonts.

Here are 4 possible solutions you may find useful. I personally use option 3.

https://github.com/github-community/community/discussions/16871#discussioncomment-2785108

### Option 1
Disable block fingerprinting at `brave://settings/shields` (https://github.com/gitpod-io/gitpod/issues/5014#issuecomment-1063944871)

### Option 2
Turn off shields for the site (it seems like this must be done for each project individually) (https://github.com/gitpod-io/gitpod/issues/5014#issuecomment-947238614)

### Option 3
Set `terminal.integrated.gpuAcceleration` to `canvas` in the settings (https://github.com/gitpod-io/gitpod/issues/5014#issuecomment-919901497)

```
"terminal.integrated.gpuAcceleration": "canvas"
```

### Option 4
Adjust the `terminal.integrated.lineHeight` and `terminal.integrated.letterSpacing` settings (https://github.com/gitpod-io/gitpod/issues/5014#issuecomment-891781810)

```
"terminal.integrated.lineHeight": 1.2,
"terminal.integrated.letterSpacing": 2
```

**Reply by EduApps-CDG** [reply]
If I change the GPU acceleration to canvas, will it have some sort of performance difference?

**Reply by DenverCoder1** [reply]
I'm not sure. It seems fine when I use it, but you can try it and see how well it works for you.
